examples/pytorch/eeg-gcnn/main.py

```diff
@@ -149,7 +155,7 @@ def _load_memory_mapped_array(file_name):
     # Dataloader========================================================================================================
 
     # use WeightedRandomSampler to balance the training dataset
-    NUM_WORKERS = 4
+    _NUM_WORKERS = args.num_workers
```

Comment:
it's a local variable, let's name it as `num_workers`. Or just assign with `args.num_workers` directly.

Reply:
I was actually trying to follow some [pattern](https://github.com/dmlc/dgl/blob/0eb56da2a0afa1138afdeb086d78de3f5407c728/examples/pytorch/eeg-gcnn/main.py#L102-L104) that was used in this file.
In any case, it's done.